packages/odyssey-storybook/src/utilities/Utilities.stories.mdx

```diff
@@ -0,0 +1,202 @@
+
+This component uses an `as="abbr"` polymorphic API to keep the DOM a11y compliant as well as clean and comprehensible by developers and their automated tests.
+
+`<Box />` contains a minimal set of styles that are provided through inheritance to children (`font-family`) and provides an API for utility tailwind-esque styling (`padding`, `margin`).
+
+These styles that are owned by `<Box />` are also a centralized place to apply theme-able odyssey design tokens for override (theming `font-family` "globally").
+
+Going further, `<Box />` accepts a `className` prop for applying component-specific styling. This prevents the need for `<Box />` to define every style for all use cases. Additionally, parent component classes can contain their own theme-able tokens to keep things customizable.
+
+A simple component like `<Button>` might look like this:
+
+```html
+<Box {...omitProps} ref="{ref}" as="button" className="{componentClass}">
+  { label }
+</Box>
+```
+
+**WIP API NOTE:** Can Box handle spreading props and the ref under the hood, so component authors wouldn't need to specify?
+
+#### Global styles
+
+Box includes these inheritable defaults:
+
+- color: $text-body;
+- font-family: $body-font-family;
+- font-style: normal;
+- font-weight: 400;
+- letter-spacing: 0;
+- line-height: $base-line-height;
+- font-feature-settings: "lnum", "pnum";
+- text-decoration-skip-ink: all;
+
+Box includes these parent resets:
+
+- box-sizing: border-box;
+- margin-block: 0;
+- margin-inline: 0;
+- padding-block: 0;
+- padding-inline: 0;
+- border: 0;
+- font: inherit;
+- font-size: 100%;
+- vertical-align: baseline;
+
+Component-specific styling may override these defaults via `className`, as above.
+
+Authors may specify in-situ variants to reset properties via utility props.
+
+### Text
+
+**WIP API NOTE:** Would anyone be opposed to renaming this component to `<StyledElement>` or similar?
+
+`<Text />` is a convenience utility built with `<Box />`. It can provide its own hooks for customization and theme-able tokens as necessary.
+
+#### Styled element insertion
+
+`<Text />` enables authors and components to insert styled HTML5 elements without the need for a global CSS sheet.
+
+```html
+We <Text as="em">really</Text> need to get off this planet.
+```
+
+A more complicated example might look like this:
+
+```html
+<Text as="blockquote">
+  <Text as="p">
+    The important thing is not to stop questioning. Curiosity has its own reason
+    for existence.
+  </Text>
+  <footer>
+    Albert Einstein,
+    <Text as="cite">
+      Old Man's Advice to Youth: "Never Lose a Holy Curiosity," LIFE magazine (2
+      May 1955) statement to William Miller, p. 64.
+    </Text>
+  </footer>
+</Text>
+```
+
+####
+
+**WIP API NOTE:** This next section is an additional use-case for `<Text />`.
+
+In addition, `<Text />` provides a simpler API with indirect descendant selectors for natural HTML5 composition or styling dynamic content (e.g. from markdown).
```

Comment:
we should avoid selling this as a solution for styling dynamic content - it only handles intrinsic text elements today and we don't have plans to change that soon. Lets remove that doc section but I am glad you included the code example to contrast against the one above.